WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -781,6 +782,19 @@ - (void)showFailedMediaBeforeEditAlert
     [self presentViewController:alertController animated:YES completion:nil];
 }
 
+#pragma mark - Toolbar options
+
+- (void)sharePost
+{
+    if ([self.post isKindOfClass:[Post class]]) {
```

Comment:
Same as above for this.  Early returns are just fine for longer methods, but don't really see them as positive for smaller methods such as this one (the method ends up having an extra line of code for no gain).